code/datums/components/mind_linker.dm

```diff
@@ -186,15 +186,15 @@
 	background_icon_state = linker.speech_action_background_icon_state
 
 /datum/action/innate/linked_speech/IsAvailable()
-	return ..() && (owner.stat != DEAD)
+	return ..() && (owner.stat <= UNCONSCIOUS)
```

Comment:
why equals

Reply:
Autism

It seemed to still work as intended somehow but ill fix and test tomorrow.